_posts/2020-02-08-cfp-resources.md

```diff
@@ -0,0 +1,209 @@
+
+<br/>
+
+###  Lengths of talks.
+
+For the 2020 event - you can submit for three different lengths of talks.
+
+- **60-minute talks:** This is the same as the talks from last year. We suggest a talk length of 45 - 50 minutes and time for questions. If you do not wish to take questions, please aim to have the talk adaptable so that 10 - 15 minutes can be cut if needed. This is only in case of emergencies, such as the fire alarms in 2019 which resorted in the schedule having to be altered. These will have their own slots with breaks before and after.
+
+- **30-minute talks:** These talks are shorter and can be a 30 full minutes or 20 - 25 minutes with the remaining time for questions from the audience. These will have their own slots with breaks before and after.
+
+- **15-minute lightning talks:** If you think a 30-minute talk is a bit too intimidating, how about submitting for a lightning talk? Three lightning talks will be presented in a session, one after the other with breaks before the first talk and after the third.
+
+<br/>
+
+###  Who can submit?
+Anybody working in or learning about technology! No matter what your discipline or level of experience, you can submit to DDD East Midlands conference.
+
+Designers, developers, managers, founders, user researchers and more! We want submissions from you ALL.
+
+<br/>
+
+###  What sort of subjects are you looking for?
+Again, anything relevant to the technology industry. Examples of topic areas include:
+
+- **Tech Talks:** Anything from "a beginners guide to" to war stories and deep dives. 
+
+- **Core Skills:** Some people refer to these as "soft skills" or "people skills". Talks relating to that skill set we all require - management, ethics, people and more.
+
+- **Other** Maybe you have experience which means you could talk about another perspective of tech. Legalities, finances or otherwise. You might have a topic that you think would benefit a broad range of technologists.
+
+<br/>
+
+###  What is the "anonymous submission" about?
+
+Talks are selected via attendee vote. Attendees are only shown talk titles and descriptions (which is why we ask that these do not contain any identifiable information).
+
+There is more information about the anonymous submission process, and why we run this process <a href="https://blog.dddeastmidlands.com/2019/12/07/submission-process.html" target="_blank">int this blog post: The Anonymous Submission Process.</a>
+
+<br/>
+
+###  Who attended last year?
+
+To find out more about last years attendees, check out our previous post <a href="https://blog.dddeastmidlands.com/2020/01/26/stats-from-2019.html" target="_blank">Statistics from 2019.</a>
+
+<br/>
+
+###  Is there any information on last years submissionS?
```

Comment:
```suggestion
###  Is there any information on last years submissions?
```

Reply:
@wordshaker looks like we'd forgotten to get this one in - would you mind raising a PR for it?